examples/angular-example/src/app/app.component.ts

```diff
@@ -65,7 +66,7 @@ export class AppComponent implements OnInit {
     },
   }
 
-  uppy: Uppy = new Uppy({ debug: true, autoProceed: true })
+  uppy: Uppy<Meta, Body> = new Uppy({ debug: true, autoProceed: true })
```

Comment:
This doesn't do it for now?
```suggestion
  uppy: Uppy<any, any> = new Uppy({ debug: true, autoProceed: true })
```

Reply:
Yeah, it forces us to silence the `no-explicit-any` rule – and I wouldn't be surprised if some users prefer to depend on `@uppy/utils` rather than use `any` – but it's probably better for now.